packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -388,19 +392,29 @@ class AppBarState extends State<AppBar> {
       );
     }
     if (config.actions != null && config.actions.isNotEmpty) {
+      // Expand to at least 48dp if IconButton or PopupMenuButton.
+      List<Widget> sizedActions = config.actions.map<Widget>((Widget action) {
+        return action is IconButton || action is PopupMenuButton ?
```

Comment:
Runtime type checks like these break composibility.  For example, if `action` is a widget that builds an `IconButton`, it will work differently than a straight `IconButton`.  Similarly, if you copy and paste the code for `IconButton` into your own codebase and change its name to `IconButton2`, then it will behave differently.

Instead, we should just apply this constraint to all the children, regardless of what type there are.  That is to say, AppBar should have the opinion that the actions have a min width of 48.0.

Also, we tend not to use `map` because it's less efficient than just writing out the loop.